book/login/user_account_component.md

```diff
@@ -0,0 +1,36 @@
+# User Account Component (Login/Logout)
+
+## Overview
+
+The User Account Component is a UI component that can be added to screens via Zapp Studio (aka the UI builder) to trigger the Login / Logout functionality of the Login Plugin used by the app.
+The typical usage of this component is on a Settings screen, where the app lets the user login to / logout out their account.
+
+Login plugin developers should follow the implementation notes below to make sure their login plugin is compatible with the User Account Component.
+
```

Comment:
@guysarfatty you have unneeded line space here.

Reply:
I checked the looks on gitbook, it doesn't create an extra spacing there...